packages/core/src/routes/admin-user.ts

```diff
@@ -122,10 +125,146 @@ export default function adminUserRoutes<T extends AuthedRouter>(router: T) {
     }
   );
 
+  router.patch(
+    '/users/:userId/password',
+    koaGuard({
+      params: object({ userId: string() }),
+      body: object({ password: string().min(6) }),
```

Comment:
maybe add comments to demonstrate the rules of setting a proper/valid password?
the rule can be the length of the password is greater than or equal to six for now, but can be customized in the future or per users' own preferences.

Reply:
good